src/security.tf

```diff
@@ -156,14 +156,15 @@ resource "azurerm_key_vault_access_policy" "adgroup_security_policy" {
 
 ## azure devops ##
 data "azuread_service_principal" "azdo_sp_tls_cert" {
+  count        = var.azdo_sp_tls_cert_enabled ? 1 : 0
   display_name = format("azdo-sp-%s-tls-cert", local.project)
 }
 
 resource "azurerm_key_vault_access_policy" "azdo_sp_tls_cert" {
   count        = var.azdo_sp_tls_cert_enabled ? 1 : 0
   key_vault_id = module.key_vault.id
   tenant_id    = data.azurerm_client_config.current.tenant_id
-  object_id    = data.azuread_service_principal.azdo_sp_tls_cert.object_id
+  object_id    = data.azuread_service_principal.azdo_sp_tls_cert.*.object_id
```

Comment:
```suggestion
  object_id    = data.azuread_service_principal.azdo_sp_tls_cert[0].object_id
```

Reply:
I quite sure that with star sign ( * ) it fails since it's going to return a list .... while only one object id is expected here.

Reply:
Thank you for the review. I've tried the plan both with `.*.` and `[0]`: the output is the same, but quite strangely the resource `azdo_sp_tls_cert` is not marked `updated in place` in neither case. Anyway please feel free to commit your suggestion. We should postpone the `apply` of this PR in production in the afternoon if you agree, since at 12.00 we should monitor CONSAP flow and we need the system to be up and running

Reply:
@uolter it's correct, I tested a `plan` in `dev` and `prod` env.
`azdo_sp_tls_cert` is not marked as `updated in place` because:
- dev: object_id not changes
- prod: `azdo_sp_tls_cert` not exixts